pkg/minibroker/minibroker.go

```diff
@@ -88,12 +86,12 @@ type Client struct {
 	serviceCatalogEnabledOnly bool
 }
 
-func NewClient(repoURL string, serviceCatalogEnabledOnly bool) *Client {
+func NewClient(serviceCatalogEnabledOnly bool) *Client {
 	klog.V(5).Infof("minibroker: initializing a new client")
 	return &Client{
-		helm:                      helm.NewClient(repoURL),
+		helm:                      helm.NewDefaultClient(),
 		coreClient:                loadInClusterClient(),
-		namespace:                 loadNamespace(),
+		namespace:                 os.Getenv("NAMESPACE"),
```

Comment:
We should not be reading environment variables from a constructor. I need to refactor this.

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/72/commits/2d929568e1a1995f9f249c70c049ceb2e3b9f9c3.